bigquery/google/cloud/bigquery/external_config.py

```diff
@@ -41,49 +37,100 @@ class BigtableColumn(object):
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.qualifierEncoded
-    """
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.fieldName
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.fieldName
+        """
+        prop = self._properties.get('fieldName')
+        return prop
 
-    qualifier_string = _TypedApiResourceProperty(
-        'qualifier_string', 'qualifierString', six.string_types)
-    """See
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.qualifierString
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.qualifierString
-    """
+    @field_name.setter
+    def field_name(self, value):
+        self._properties['fieldName'] = value
 
-    type_ = _TypedApiResourceProperty('type_', 'type', six.string_types)
-    """See
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.type
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.type
-    """
+    @property
+    def only_read_latest(self):
+        """bool: If this is set, only the latest version of value in this
+        column are exposed.
+
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.onlyReadLatest
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.onlyReadLatest
+        """
+        prop = self._properties.get('onlyReadLatest')
+        return prop
+
+    @only_read_latest.setter
+    def only_read_latest(self, value):
+        self._properties['onlyReadLatest'] = value
+
+    @property
+    def qualifier_encoded(self):
+        """Union[str, bytes]: The qualifier encoded in binary.
+
+        The type is ``str`` (Python 2.x) or ``bytes`` (Python 3.x). The module
+        will handle base64 encoding for you.
+
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.qualifierEncoded
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.qualifierEncoded
+        """
+        prop = self._properties.get('qualifierEncoded')
```

Comment:
Do we need to base64 decode here? (Note if we do it here, we'll also have to handle `None`)